packages/rke2-calico/generated-changes/patch/values.yaml.patch

```diff
@@ -17,15 +17,26 @@
 +    bgp: Disabled
 +    ipPools:
 +    - natOutgoing: Enabled
++      blockSize: 24
 +      encapsulation: VXLAN
 +      cidr: 10.42.0.0/16
 +  imagePath: "rancher"
 +  imagePrefix: "mirrored-calico-"
 +
++felixconfigurations:
++  spec:
++    allowVXLANPacketsFromWorkloads: true
++    vxlanEnabled: true
++    ipipEnabled: false
++    vxlanVNI: 4096
++
++ipamconfigs:
++  spec:
++    strictAffinity: true
```

Comment:
Unfortunately, I don't think you can pass these parameters to the operator because it does not understand them. IIRC, the current operator of Tigera has no way to fill a felixconfiguration or an ipamconfig ==> https://docs.projectcalico.org/reference/installation/api#operator.tigera.io/v1.Provider